src/main/java/seedu/duke/Ui.java

```diff
@@ -50,7 +50,7 @@ public static void getUserCommand(EventList eventList) {
      * Prints a welcome message for users when application is launched
      */
     public static void showWelcome() {
-        final String LOGO = ("888b      88  88        88   ad88888ba   88888888ba   88\n"
+        String logo = ("888b      88  88        88   ad88888ba   88888888ba   88\n"
```

Comment:
As per coding standards, final variable should be all caps. See [this](https://nus-cs2113-ay2223s2.github.io/website/schedule/week4/topics.html#w4-3-java-constants)

Reply:
Thank you for spotting this!